src/test/resources/biomes/1.21.x.json

```diff
@@ -0,0 +1,67 @@
+[
+    "minecraft:ocean",
```

Comment:
no idea if this is right or how to even gen this

we need a `make biome-list` or some shiz

Reply:
I think MockBukkit uses MetaMiner to grab the files from the server jar. Usually I just manually yeet from their repo, but we can look into how they use it. I will say that their tooling is all gradle though so we might not even be able to reuse their tooling in maven. If we can we should.